docs/_documentations/importing-existing-projects.md

```diff
@@ -93,30 +93,30 @@ Configure your `pom.xml` file as follows:
   </profile>
   ```
 
-  Then add the required Liberty Maven plugin configuration:
+3. Add the required Liberty Maven plugin configuration:
 
-  2b. Liberty server configuration file, `server.xml`, that is located in the source folder that is referenced in the `pom.xml` file.
+- Liberty server configuration file, `server.xml`, that is located in the source folder that is referenced in the `pom.xml` file.
 
   ```xml
   <configFile>${basedir}/src/main/liberty/config/server.xml</configFile>
   ```
 
-  2c. Optional: Liberty server environment file:
+- Optional: Liberty server environment file:
```

Comment:
In line 104 and in line 110, I think users will understand that "Add" is the implied verb here. For added clarity, though, I'd go ahead and include the verb "Add."
- Add the Liberty server...
- Add the Liberty Java...